skyportal/app_server.py

```diff
@@ -86,6 +87,35 @@
 
 log = make_log('app_server')
 
+
+class CustomApplication(tornado.web.Application):
+    def log_request(self, handler):
+        """Writes a completed HTTP request to the logs.
+        By default writes to the python root logger.  To change
+        this behavior either subclass Application and override this method,
+        or pass a function in the application settings dictionary as
+        ``log_function``.
+        """
+        if "google-oauth2" in str(handler._request_summary()):
```

Comment:
I'm not confident about this method; what code is this based on?  It feels a bit magical.  What does this do that we need, that is not currently being done?

Reply:
@stefanv We've discussed this. You said you don't want to see the "access error 500" messages in the logs when they're associated with the PSA pipeline -- this filters those out (the URL always contains "google-oauth2" in those cases). What's magical about it? I'm happy for you to propose an alternate implementation.

Reply:
Looks good now!  Please also add a comment to explain why we do this.

Reply:
The comment says "message text" and the comment above says "in the request route".  Request route makes the most sense to me; is `_request_summary()` (private method?) the best way to get to that?

Reply:
Comment updated. Yes, why is `_request_summary` not a good way to get that?